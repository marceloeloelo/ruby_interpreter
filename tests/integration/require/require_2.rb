require "tests/integration/require/required_file.rb"
require "tests/integration/require/required_file.rb"
require "tests/integration/require/required_file.rb"

puts ("This is 'require_2.rb'.")
