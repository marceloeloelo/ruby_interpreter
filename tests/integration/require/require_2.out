This is 'included_file.rb'
This is 'require_1.rb'
